### PR TITLE
Sb order confirmation/purchased items widget

### PR DESCRIPTION
**Pull request by Tim-M-S**
### All changes meet the following requirements
~~[ ] Changelog entry was added~~
- [x] Changes have been tested
- [x] Plugin can be built

@plentymarkets/ceres-io

### CODE REVIEW

File: resources/lang/en/Template.properties [hunk 1]
```diff
@@ -468,6 +468,7 @@ orderConfirmationPaymentStatus_overpaid = "Overpaid"
 orderConfirmationPaymentStatus_partlyPaid = "Partially paid"
 orderConfirmationPaymentStatus_prepaid = "Prepaid"
 orderConfirmationPaymentStatus_unpaid = "Not paid"
+orderConfirmationProperties = "Order properties"
```
Comment (fmutschler) [inline resources/lang/en/Template.properties]:
```suggestion
orderConfirmationProperties = "Order characteristics"
```

File: resources/lang/en/Widget.properties [hunk 2]
```diff
@@ -735,3 +735,5 @@ googleMapsPlaceholderTitle = "No active map"
 googleMapsPlaceholderDescription = "The map won't be displayed until the Google API key and an address have been entered."
 ; AcceptPrivacyPolicyWidget
 acceptPrivacyPolicyLabel = "Privacy policy checkbox"
+; ItemsBoughtWidget
+itemsBoughtLabel = "Items bought"
```
Comment (fmutschler) [inline resources/lang/en/Widget.properties]:
```suggestion
itemsBoughtLabel = "Purchased items"
```

**Review by fmutschler — CHANGES_REQUESTED** [review]

File: resources/lang/en/Template.properties (hunk 1 above)
Comment (Tim-M-S) [reply resources/lang/en/Template.properties]: done

File: resources/lang/en/Widget.properties (hunk 2 above)
Comment (Tim-M-S) [reply resources/lang/en/Widget.properties]: done

**Comment by Tim-M-S** [comment]
closed

**Pull request closed without merging**